### PR TITLE
Remove reference to RTD

### PR DESCRIPTION
**Pull request by kclowes**
## What was wrong?
There was a stray Read the Docs reference in the Pull Request Template. 

## How was it fixed?

Removed it. 

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [x] Clean up commit history

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

**Comment by kclowes** [comment]
@fselmo I also made some tweaks to the isort config in the second commit, mostly because I wanted to make sure that these pass irl, before I stick them in the template.

File: tox.ini [hunk 1]
```diff
@@ -6,9 +6,10 @@ envlist=
 [isort]
 combine_as_imports=True
 force_sort_within_sections=True
+force_grid_wrap=1
 known_third_party=hypothesis,pytest
-profile=black
```
Comment (fselmo) [inline tox.ini:10]: nit: Is moving this necessary? might just messy up some commit history. I'm fine with it though.

**Review by fselmo — APPROVED** [review]
lgtm, I like the config approach. Just added the tiniest of nits which is fine if left alone too 😄

File: tox.ini (hunk 1 above)
Comment (kclowes) [reply tox.ini:10]: Ha, nope. Not sure why that got moved 🤔

**Pull request merged**